client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,47 @@
+import { useAuth } from '../../context/useAuthContext';
+import useStyles from './useStyles';
+import { Button, Card, Typography, Avatar, CardActions, Paper } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+const ProfilePhoto = (): JSX.Element => {
```

Comment:
It looks like some of this code is part of a previous PR, you may have branched off a previous branch to work on this feature. Doing so is completely fine but in the future, please base your PRs properly so that only new changes are reflected :)

Reply:
sorry for the mix up, removed